.ci_support/linux_64_.yaml

```diff
@@ -7,4 +7,4 @@ channel_targets:
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
 python_min:
-- '3.10'
+- '3.9'
```

Comment:
The bot changed the Python min version back down to 3.9... :slightly_frowning_face:. I guess there is supposed to be another way to update this somewhere.